### ISSUE TITLE
UX/UI and design wishes for accessible navigation of the book

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/the-turing-way#2034]
### Summary

* The Turing Way will work with a member of the Turing Research Engineering group who will help enhance the UX/UI of the project

### What needs to be done?

- [ ] Revamp the landing page to make navigation of the book simpler
- [ ] Build cue card feature that allows access to the curated sets of chapters (persona wise curation should be done by community members)
- [ ] Build tagging feature in The Turing Way resources (chapters, templates, pages etc.) so that it is easy to find relevant pages
- [ ] Allow people to build their personal set of curated chapter
- [ ] Design UX/UI research plan (interview, form, call for ideas, working group etc.)

### Who can help?

* REG member (6 months, half time paid position - already coordinated with the responsible group at Turing)
* Anyone who would like to share their feedback and idea should comment below

---

### Updates

-

**Comment by da5nsy** [comment]
A little piece of UI feedback: currently on mobile the hypothes.is buttons overlap with the fullscreen/github/download buttons.
I assume that this is an upstream issue though.

![Screenshot_20210723-221359](https://user-images.githubusercontent.com/3739866/126854696-bfce2584-42cd-4fda-a019-dc8d42cd4456.jpg)

**Comment by choldgraf** [comment]
Indeed that's an upstream fix. Para are most welcome - I don't understand CSS well enough to figure it out haha

**Comment by da5nsy** [comment]
Oop, @RaoOfPhysics got there before me: https://github.com/executablebooks/jupyter-book/issues/1392

**Comment by RaoOfPhysics** [comment]
@da5nsy, yeah, I posted about it in the Slack as well. :)

@choldgraf, I don't know enough CSS either, but I managed to track down a version of Jupyter Book in which the rendering is fine, but that's because the top bar is styled differently.

One option, which I proposed in https://github.com/executablebooks/jupyter-book/issues/1392 is to use a hidden sidebar.

**Comment by choldgraf** [comment]
Yeah - makes sense to try out, or at least potentially make configurable in sphinx-comments (the package that provides that functionality). I think that issue will need a champion to pick it up though, as right now I believe most people in the EBP world are totally swamped with other issues and work duties 😞

**Comment by malvikasharan** [comment]
Thanks everyone for the comments. 
I just want to emphasise that this issue is to only collect feedback (not to expect immediate solution or overwhelm the Jupyterbook developers 😅) on what needs to be improved in The Turing Way and how we can do that while ensuring that upstream contributions are made.
